### ISSUE TITLE
No error messages

### ISSUE THREAD

**Issue by Melindrea** [verbose/grunt-verb#1]
So, today I ended up having an interesting issue. I'd changed my changelog, rebuilt my docs and everything was dandy ... except, the README file wasn't updated.

Doing a few tricks I figured out it's because I don't know how to YAML (whoops ...), but it would be very useful if grunt-verb could halt or toss out errors when there are such.

**Comment by jonschlinkert** [comment]
Good to know, I'll have to think about how to best do this. l'm not sure where the error would be thrown, since data it's hard to know when data should or shouldn't be used. any thoughts?

**Comment by Melindrea** [comment]
Well, the CLI version threw some kind of error when I tried it that way.

**Comment by jonschlinkert** [comment]
hmm, maybe it's how the context is being merged. can you put some code examples of what you did wrong and what worked?

**Comment by Melindrea** [comment]
The error was that my YAML file had a comma where it oughtn't.

I have a fairly standard .verbrc.md file with `<%= changelog() %>`. 
Running `grunt verb` I get the following:

```
Running "verb:readme" (verb) task

Done, without errors.
```

However, no new README.md file was generated. Fully removing grunt verb 
and running `verb` I got the following:

```
unresolved [ 'verbfile.coffee' ]
   verb-cli [reading] · .verbrc.md

/home/marie/projects/grunt/grunt-epub/node_modules/verb/node_modules/fs-utils/index.js:223
     throw err;
           ^
YAMLException: Failed to parse 
"/home/marie/projects/grunt/grunt-epub/CHANGELOG": JS-YAML: bad 
indentation of a mapping entry at line 2, column 23:
         date: "2014-05-06",
```

My Gruntfile doesn't have any particular config for verb, using the 
standard one.

**Comment by doowb** [comment]
Looks like the [verb-cli loads data with plasma](https://github.com/assemble/verb-cli/blob/master/lib/tasks.js#L55) automatically, but I don't see that in grunt-verb so this could be the reason no error was being thrown.

After you fixed it the data, does it create the README.md file correctly?

**Comment by Melindrea** [comment]
Yep, once I figured out the error and fixed my YAML file everything was 
dandy.

**Comment by jonschlinkert** [comment]
ok, that was what I was wondering but didn't have a chance to check. so @doowb it sounds like plasma is throwing the error, which is a good thing. gives us a starting point, thanks
